ui/Screen/Project/Source/PatchList.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="typescript">
+  import * as router from "ui/src/router";
+
+  import type { Project } from "ui/src/project";
+  import type { Patch } from "ui/src/project/patch";
+  import {
+    EmptyState,
+    List,
+    SegmentedControl,
+  } from "ui/DesignSystem/Component";
+  import PatchCard from "./PatchCard.svelte";
+
+  export let patches: Patch[];
+  export let project: Project;
+  export let filter: "open" | "closed" | "all";
+
+  const defaultBranch = project.metadata.defaultBranch;
+
+  const selectPatch = ({ detail: patch }: { detail: Patch }) => {
+    router.push({
+      type: "project",
+      urn: project.urn,
+      activeView: {
+        type: "patch",
+        peerId: patch.peerId,
+        id: patch.id,
+      },
+    });
+  };
+
+  const filterOptions = [
+    {
+      title: "Open",
+      value: "open",
+    },
+    {
+      title: "Closed",
+      value: "closed",
+    },
+    {
+      title: "All",
+      value: "all",
+    },
+  ];
+
+  let filteredPatches: Patch[];
+  $: {
+    switch (filter) {
+      case "open":
+        filteredPatches = patches.filter(patch => !patch.merged);
+        break;
+      case "closed":
+        filteredPatches = patches.filter(patch => patch.merged);
+        break;
+      case "all":
+        filteredPatches = patches;
+        break;
+      default:
+        console.log(filter);
```

Comment:
👀

Reply:
So strange that eslint didn’t catch that

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1915/commits/611d187a74e5057512e9f0220b9cd0622e56658c